.env.example

```diff
@@ -13,7 +13,6 @@ DATABASE_URL=postgis://postgres:postgres@db:5432/postgres
 DATABASE_URL_HUB_MIS=postgis://postgres:postgres@db:5432/mis_datahub
 DATABASE_URL_HUB_CA=postgis://postgres:postgres@db:5432/ca_datahub
 DATABASE_URL_HUB_ERP=postgis://postgres:postgres@db:5432/erp_datahub
-DATABASE_URL_HUB_REGISTRATION=postgis://postgres:postgres@db:5432/rdi_datahub
```

Comment:
this should be removed in the next pr after removing models from registration_datahub